.travis.yml

```diff
@@ -37,10 +36,7 @@ jobs:
     - stage: tests
       name: 'Integration Tests Linux'
       script:
-        # cannot use 'make test-integration' because 'make clean' fails with
```

Comment:
Why did you remove this? `make clean` and `make test-integration` fails in my laptop with go 1.13 with the project outside of `GOPATH`.
Does it work for you outside of `GOPATH`?

Reply:
it was fixed for `1.13` inside of `$GOPATH`, but not outside :(
reverted.